src/library/scala/math/Equiv.scala

```diff
@@ -58,4 +69,6 @@ object Equiv extends LowPriorityEquiv {
     ((x, y) => implicitly[Equiv[S]].equiv(f(x), f(y)))
 
   @inline def apply[T: Equiv]: Equiv[T] = implicitly[Equiv[T]]
+
+  implicit def equivFromPartialOrdering[T](implicit ev: PartialOrdering[T]): Equiv[T] = ev
```

Comment:
this is probably going to produce very confusing deprecation messages for `Equiv[Float]` and `Equiv[Double]`, but don't worry about that. I'll improve it in a follow-up PR. that's not your problem to have to deal with